### PR TITLE
update director charts

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- update director to the newest version

**Related issue(s)**

kyma-incubator/compass#441

### CODE REVIEW

**Comment by dbadura** [comment]
\retest

**Comment by dbadura** [comment]
\retest netlify/kyma-project-docs-preview/deploy-preview

File: tests/compass-runtime-agent/test/runtimeagent/test/runtime_agent_test.go [hunk 1]
```diff
@@ -194,14 +194,14 @@ func TestCompassRuntimeAgentSynchronization(t *testing.T) {
 				return applications.NewApplication("test-app-3", "", map[string]interface{}{}).
 					WithAPIs(
 						[]*applications.APIDefinitionInput{
-							applications.NewAPI("no-auth-api", "no auth api", testSuite.GetMockServiceURL()),
-							applications.NewAPI("basic-auth-api", "basic auth api", testSuite.GetMockServiceURL()).WithAuth(basicAuth),
-							applications.NewAPI("oauth-auth-api", "oauth api", testSuite.GetMockServiceURL()).WithAuth(oauth),
+							applications.NewAPI("no-auth-api", "no auth api", testSuite.GetMockServiceURL()).WithJsonApiSpec(&emptySpec),
+							applications.NewAPI("basic-auth-api", "basic auth api", testSuite.GetMockServiceURL()).WithAuth(basicAuth).WithXMLApiSpec(&emptySpec),
```
Comment (rafalpotempa) [inline tests/compass-runtime-agent/test/runtimeagent/test/runtime_agent_test.go]: Nice.

**Pull request merged**